skin/base.css

```diff
@@ -664,6 +664,16 @@
   transition: background-position 150ms ease-out;
 }
 
+/* Use the Container color if available */
+.tabbrowser-tab[usercontextid] {
+  --tab-highlight-color: var(--identity-tab-color);
+}
+
+/* Make the indicator partly visible for Container tabs */
+.tabbrowser-tab[usercontextid] .tab-selected-indicator:not([pinned]) {
```

Comment:
The `.tab-selected-indicator` will never have the `pinned` attribute. If you are trying to not affect pinned tabs this should be on the `.tabbrowser-tab` selector.